libshaderc/include/shaderc.h

```diff
@@ -113,7 +113,7 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
-// Set the compiler to emit a disassembly text instead of a binary. In
```

Comment:
I'm not sure why this shows up as a diff in your PR.  I'd have expected it to be the same on both sides.